package.json

```diff
@@ -68,7 +68,7 @@
     "typescript": "^4.0.3"
   },
   "peerDependencies": {
-    "react": "^16.8.6"
+    "react": ">=16.8.6"
```

Comment:
Could you please change the React version to `^16.8.0 || ^17.0.0"`? Think this would be a better practice since we don't know yet if React 18 will work with this lib.

Reply:
I have updated. Yes, true, 18.0 is unsafe to assume compliant.